kobo/client/commands/cmd_resubmit_tasks.py

```diff
@@ -33,7 +36,7 @@ def run(self, *args, **kwargs):
         failed = False
         for task_id in tasks:
             try:
-                resubmitted_id = self.hub.client.resubmit_task(task_id, kwargs.pop("force", False))
+                resubmitted_id = self.hub.client.resubmit_task(task_id, *[arg for arg in [force, priority] if arg])
```

Comment:
I think this will break if a user uses --priority and not --force.  Then the sequence of arguments will work out to:

```
  resubmit_task(task_id, priority)
```

...which is not what the server expects.

Reply:
Right, fixed.